exercises/clock/clock_test.go

```diff
@@ -7,7 +7,7 @@ import (
 
 // Clock type API:
 //
-// Time(hour, minute int) Clock    // a "constructor"
+// New(hour, minute int) Clock    // a "constructor"
```

Comment:
I know this is super nit, but notice how this `// a "constructor"` was aligned with the `// a "stringer"` comment below, but now it's misaligned by one space because of `Time` -> `New`. Perhaps add a space back?